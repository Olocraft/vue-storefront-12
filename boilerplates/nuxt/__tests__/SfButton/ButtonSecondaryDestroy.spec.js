import { shallowMount } from '@vue/test-utils'
import ButtonDestroySecondary from '../../components/SfButton/Secondary/Destroy/ButtonSecondaryDestroy.vue'

let wrapper

describe('ButtonDestroySecondary.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(ButtonDestroySecondary)
  })
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
