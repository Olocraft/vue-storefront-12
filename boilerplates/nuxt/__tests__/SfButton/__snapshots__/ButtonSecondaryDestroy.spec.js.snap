// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonDestroySecondary.vue render component correctly 1`] = `<button class="inline-flex items-center justify-center cursor-pointer border border-rose-400 disabled:border-0 bg-white hover:bg-rose-100 active:bg-rose-200 disabled:bg-gray-200 text-rose-600 hover:text-rose-600 active:text-rose-700 disabled:text-gray-500 disabled:opacity-50 uppercase font-body text-base font-medium rounded-md px-[16px] py-2 shadow hover:shadow-md disabled:shadow-none disabled:cursor-not-allowed outline-violet"></button>`;
